### PR TITLE
Ra 89

### PR DESCRIPTION
**Pull request by aameijide**

### CODE REVIEW

**Comment by ylatuya** [comment]
That's already handled automatically by gtk with this configuration file:
/Applications/LongoMatch\ PRO.app/Contents/Home/share/themes/Mac/gtk-2.0-key/gtkrc

File: VAS.Tests/Core/Common/TestStateController.cs [hunk 1]
```diff
@@ -35,7 +36,7 @@ public void InitializeStateController ()
 		{
 			sc = new StateController ();
 			screenStateMock = new Mock<IScreenState> ();
-			screenStateMock.Setup (x => x.PreTransition (It.IsAny<object> ())).Returns (AsyncHelpers.Return (true));
+			screenStateMock.Setup (x => x.PreTransition (It.IsAny<ExpandoObject> ())).Returns (Task.Factory.StartNew (() => true));
```
Comment (aoller) [inline VAS.Tests/Core/Common/TestStateController.cs]: Use AsyncHelpers

**Comment by vguzmanp** [comment]
:+1:

**Comment by aameijide** [comment]
All ``if (((int)evt.State & (int)Gdk.ModifierType.ShiftMask) == (int)Gdk.ModifierType.ShiftMask)``can be changed for ``evt.State.HasFlag(Gdk.ModifierType.ShiftMask)``

**Comment by vguzmanp** [comment]
:+1:

**Pull request merged**